### ISSUE TITLE
Advertising is not blocked at a specific app.

### ISSUE THREAD

**Issue by Richelieu4771** [AdguardTeam/AdguardForiOS#1769]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ ] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Issue Details

- **AdGuard version:** 
  - AdGuard for iOS 4.2.0(699) 
- **Device model and storage size:**
  -  iPhone 12 Pro Max 256GB 
- **Operating system and version:** 
  - iOS 14.5.1 
- **Browser or App:** 
  - Kakaotalk (https://apps.apple.com/kr/app/%EC%B9%B4%EC%B9%B4%EC%98%A4%ED%86%A1-kakaotalk/id362057947)

### Expected Behavior
Advertising should be blocked and empty or equivalent.

### Actual Behavior
Advertising still exposed.

### Screenshots

<details><summary>Screenshot: AG executed in full tunnel mode. The window showing that there is no advertisement.</summary>

![KakaoTalk_20210602_191142784](https://user-images.githubusercontent.com/68171816/120463206-a430a080-c3d6-11eb-95d5-aa7a0e32a7ce.jpg)

</details>

<details><summary>Screenshot: AG executed in integrated mode. Advertising is being expressed.</summary>

![KakaoTalk_20210602_184531956](https://user-images.githubusercontent.com/68171816/120463467-e659e200-c3d6-11eb-80bd-96b4e3b7d6d7.jpg)

</details>

<details><summary>Screenshot: AG executed in integrated mode. I intentionally tried to watch it on an app that allows me to watch video ads to delete ads for a certain period of time.</summary>

![KakaoTalk_20210602_184539066](https://user-images.githubusercontent.com/68171816/120463533-f70a5800-c3d6-11eb-8dbe-a76683851720.jpg)

</details>
### Additional Information
  - I tried to block almost everything related to the app by looking at AG's filtering log to clear the ad screen, but it was exposed like the screenshot, and I found it impossible with my power when I saw it working well in normal mode. 

P.S. I'm sorry this issue was uploaded in duplicate to the Filters issue. This doesn't seem to work by making a Filter list. In integrated mode, advertisements have already been blocked by a filter called List-KR, but have been expressed.

**Comment by zzebrum** [comment]
@Richelieu4771 1) what DNS server and DNS subscription do you use?
2) when the filtering problem begin?

**Comment by Richelieu4771** [comment]
@zzebrum 1) I'm currently using Cloudflare Secure DNS, but the same thing happened when I used AdGuard DNS. AdGuard SDNS filter and List-KR were applied together on either side DNS server.
2) The problem occurred when I updated it to iOS 14.5 while using AdGuard 4.1.1. Previously, AdGuard and other VPNs could be run together to block ads.

**Comment by zzebrum** [comment]
@Richelieu4771 then it is a duplicate of [#1755](https://github.com/AdguardTeam/AdguardForiOS/issues/1755#issuecomment-851887267) - there is a bug in iOS :(
